bundles/EcommerceFrameworkBundle/FilterService/FilterType/MultiSelect.php

```diff
@@ -16,18 +16,24 @@
 
 use Pimcore\Bundle\EcommerceFrameworkBundle\IndexService\ProductList\ProductListInterface;
 use Pimcore\Bundle\EcommerceFrameworkBundle\Model\AbstractFilterDefinitionType;
+use Pimcore\Db;
 
 class MultiSelect extends AbstractFilterType
 {
     public function getFilterValues(AbstractFilterDefinitionType $filterDefinition, ProductListInterface $productList, array $currentFilter): array
     {
         $field = $this->getField($filterDefinition);
 
+        $useAndCondition = false;
```

Comment:
we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiSelect`, other wise throw `InvalidConfigException`

Reply:
still open